### DISCUSSION TITLE
Error loading Numpy

### DISCUSSION THREAD

**Discussion by Andyk1234** [pyscripter/python4delphi#448] [Q&A]
Sorry if this is a dumb question as I'm new to python and trying to learn how to integrate it into my delphi apps as a scripter.

if I put a single command in a script

import numpy

which I know is wrong, but is it normal for this to produce a delphi floating point divide by zero error?

**Comment by pyscripter** [comment]
See https://github.com/pyscripter/python4delphi/wiki/MaskFPUExceptions.
